include/seqan3/alphabet/composite/alphabet_tuple_base.hpp

```diff
@@ -26,6 +26,8 @@
 #include <seqan3/core/concept/tuple.hpp>
 #include <seqan3/core/type_traits/transformation_trait_or.hpp>
 #include <seqan3/utility/detail/integer_traits.hpp>
+#include <seqan3/utility/type_list/traits.hpp>
+#include <seqan3/utility/type_pack/traits.hpp>
```

Comment:
Why do you need to add those here when there were not included before?

Reply:
Because the file is using both of them. Before it was getting it indirectly through some other include, which itself didn't used them. So by removing it, it appeared that this header is the one that actually needs it.